src/app/shared/entry/base-url.pipe.spec.ts

```diff
@@ -5,7 +5,9 @@ describe('Pipe: BaseUrl', () => {
     const pipe = new BaseUrlPipe();
     expect(pipe).toBeTruthy();
     expect(pipe.transform('https://github.com/foobar/hello_world', 'master')).toBe('https://github.com/foobar/hello_world/blob/master/');
-    expect(pipe.transform('https://gitlab.com/foobar/hello_world', 'develop')).toBe('https://gitlab.com/foobar/hello_world/blob/develop/');
+    expect(pipe.transform('https://gitlab.com/foobar/hello_world', 'develop', 'dir/readme.md')).toBe(
```

Comment:
Based on this other [comment thread](https://github.com/dockstore/dockstore-ui2/pull/1753#discussion_r1164818203), I thought the readme path always started with a slash?

Reply:
Yes, it does. Sorry looks like I didn't add it on this. Thanks for catching that